src/FixedDataTable.js

```diff
@@ -368,6 +368,7 @@ var FixedDataTable = createReactClass({
       this._onScroll,
       this._shouldHandleTouchX,
       this._shouldHandleTouchY
```

Comment:
Needs a comma at the end of this line, that's why the build is failing on TravisCI.